website/docs/docs/deploy/deployment-overview.md

```diff
@@ -1,36 +1,129 @@
 ---
 title: "Deploy dbt jobs"
 id: "deployments"
-sidebar: "About job deployments"
+sidebar: "Use dbt Cloud's capabilities to run and seamlessly schedule a dbt job in production."
+hide_table_of_contents: true
+tags: ["scheduler"]
 ---
 
-Running dbt in production means setting up a system to run a _dbt job on a schedule_, rather than running dbt commands manually from the command line. Your production dbt jobs should create the tables and <Term id="view">views</Term> that your business intelligence tools and end users query. Before continuing, make sure you understand dbt's approach to [managing environments](/docs/collaborate/environments/environments-in-dbt).
+Use dbt Cloud's capabilities to run and seamlessly schedule a dbt job in production. Rather than run dbt commands manually from the command line, you can leverage the [dbt Cloud's in-app scheduling](/docs/deploy/job-scheduler) to automate how and when you execute your dbt production jobs. 
```

Comment:
```suggestion
Use dbt Cloud's capabilities to seamlessly run a dbt job in production or staging environments. Rather than run dbt commands manually from the command line, you can leverage the [dbt Cloud's in-app scheduling](/docs/deploy/job-scheduler) to automate how and when you execute dbt. 
```